lib/registries/addon/branded/new/controller.ts

```diff
@@ -24,19 +25,27 @@ export default class BrandedRegistriesNewSubmissionController extends Controller
     @tracked selectedSchema?: RegistrationSchemaModel;
     @tracked schemaOptions: RegistrationSchemaModel[] = [];
     @tracked projectOptions: NodeModel[] = [];
+    @tracked shouldShowProjectSelect: boolean = false;
```

Comment:
Super-minor: since we're not just using this for display logic but control logic, it might make more sense to call this something like `isBasedOnProject`?